src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TabsTypes } from '../Shape/TabsTypes';
+import { Tab } from '../Tab/Tab';
+
+export class Tabs extends React.Component {
+  constructor(props) {
+    super({ props });
+
+    this.state = {
+      index: 0,
+    };
+  }
+
+  onTabSelected = indexTab => () => {
+    this.setState({
+      index: indexTab,
+    });
+  };
+
+  render() {
+    const { tabs } = this.props;
+    const { index } = this.state;
+
+    return (
+      <>
+        <div className="center">
+          <ul className="nav nav-tabs">
+            {tabs.map((heading, indexTab) => (
+              <li key={tabs.title}>
+                <Tab
+                  {...heading}
```

Comment:
```suggestion
                  title={heading.title}
```
Вроде Юра говорил, что лучше не передавать через spread, а прописывать всё отдельно

Reply:
nice sugestion ,but I expected "you'll never walk alone"😎